lib/src/components/NxProgressBar/NxProgressBar.scss

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-progress-bar {
+  display: flex;
+  flex-direction: column;
+  gap: var(--nx-spacing-2x);
+  margin: var(--nx-spacing-6x) 0;
+}
+
+.nx-progress-bar--inline-counter {
+  flex-direction: row;
+}
+
+.nx-progress-bar--inline {
+  display: inline-flex;
+  margin: 0;
+}
+
+.nx-progress-bar__counter .nx-counter {
+  margin: 0;
+}
+
+.nx-progress-bar--success {
+  color: var(--nx-swatch-green-25);
+
+  .nx-counter {
+    border-color: var(--nx-swatch-green-25);
+    background-color: var(--nx-swatch-green-95);
+  }
+}
+
+.nx-progress-bar--error {
+  color: var(--nx-swatch-red-40);
+
+  .nx-counter {
+    border-color: var(--nx-swatch-red-40);
+    background-color: var(--nx-swatch-red-95);
+  }
+}
+
+.nx-progress-bar__label {
+  align-items: center;
+  display: flex;
+  gap: var(--nx-spacing-2x);
+  grid-area: label;
+
+  .nx-icon {
+    margin: 0 0 0 var(--nx-spacing-2x);
+  }
+}
+
+$normal-height: 25px;
+
+progress.nx-progress-bar__progress[value] {
```

Comment:
Why is `[value]` part of all of these selectors?  For that matter why is `progress` part of these selectors?

Reply:
> Why is `[value]` part of all of these selectors? For that matter why is `progress` part of these selectors?

I removed progress as part of this d59382ac8739c3784a9a9d2da10103eab3e3cc9b.

I tried removing [value] before but it seems to break the progress styling 🤔

Reply:
> > I tried removing [value] before but it seems to break the progress styling thinking
> 
> I just tried and removing it made no difference, everything still seemed to work fine

Oh ops I tried removing the wrong one. I removed it as part of this commit: 6e94da26a79a5bb93d1197df875142b3456e44ac

Reply:
> Oh ops I tried removing the wrong one. I removed it as part of this commit: https://github.com/sonatype/sonatype-react-shared-components/commit/6e94da26a79a5bb93d1197df875142b3456e44ac

To be clear, I'm asking about all of them.  When I tried two days ago I removed all of them with no ill effect.

Reply:
> > Oh ops I tried removing the wrong one. I removed it as part of this commit: [6e94da2](https://github.com/sonatype/sonatype-react-shared-components/commit/6e94da26a79a5bb93d1197df875142b3456e44ac)
> 
> To be clear, I'm asking about all of them. When I tried two days ago I removed all of them with no ill effect.

Removed: c523b2aa91b4ef5936961488bee8d6307f897d41